app/uk/gov/hmrc/eusubsidycompliancefrontend/views/CreateUndertakingPage.scala.html

```diff
@@ -42,7 +42,7 @@ <h2 class="govuk-heading-m">@messages("createUndertaking.h2")</h2>
         <p class="govuk-body">@messages("createUndertaking.p-1")</p>
         <h2 class="govuk-heading-m">@messages("createUndertaking.h2-2")</h2>
         <p class="govuk-body">@messages("createUndertaking.p-2")</p>
-        <p class="govuk-body">@messages("createUndertaking.p-3-1") <a class="govuk-link" href="#">@messages("createUndertaking.p-3-2")</a> @messages("createUndertaking.p-3-3")</p>
+        <p class="govuk-body">@messages("createUndertaking.p-3-1") <a class="govuk-link" href="@controllers.routes.SignOutController.signOut()">@messages("createUndertaking.p-3-2")</a> @messages("createUndertaking.p-3-3")</p>
```

Comment:
Ii created this route specifically when a business entity removes itself from an undertakign and signs out  with a message "You have been removed from this undertaking". I don't think we would want to display this message for all the sign out pages. May be we should rename this one and create a more general sign out page. what do you think?